src/main/java/com/perennial/doctorappointmentbooking/controller/AppointmentController.java

```diff
@@ -37,52 +39,47 @@
         message = "Please upload an excel file!";
         return ResponseEntity.status(HttpStatus.BAD_REQUEST).body(new ResponseMessage(message));
     }
+
+@Autowired
+DoctorRepo doctorRepo;
+    @Autowired
+    PatientRepo patientRepo;
+    @Autowired
+    private AppointmentRepo appointmentRepo;
+
     @PostMapping("/addappointment")
-    @ResponseBody
-    public Appointment addAppointment(@RequestBody Appointment appointment)
+    private Doctor placeDoctor(@RequestBody Request request)
     {
-        return appointmentService.addAppointment(appointment);
+        return doctorRepo.save(request.getDoctor());
     }
-//
-    @GetMapping("/appointment")
-    public List<Appointment> getAllAppointment() {
-        return this.appointmentService.getAllAppointment();
+    @PostMapping("/addPatient")
+    private Patient placePatient(@RequestBody Request request)
+    {
+        return patientRepo.save(request.getPatient());
     }
 
+    @GetMapping("/findAllAppointment")
+    public List<Appointment> findAllAppointment()
+    {
+        return appointmentRepo.findAll();
+    }
 
-//    @Autowired
-//    private DoctorRepo doctorRepo;
-//    @Autowired
-//    private AppointmentRepo appointmentRepo;
-//    @PostMapping("/addAppointment")
-//    public Doctor addAppointment(@RequestBody Request request)
-//    {
-//        return doctorRepo.save(request.getDoctor());
-//    }
-//    @GetMapping("/findAllAppointment")
-//    public List<Doctor> findAllAppointment()
-//    {
-//        return doctorRepo.findAll();
-//    }
-//
+@GetMapping("/getAppointmentById/{id}")
```

Comment:
Follow API design guidelines use - as word separators. Do not use camel case in the URL names